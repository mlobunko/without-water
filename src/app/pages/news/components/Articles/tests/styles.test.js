import React from 'react';
import { shallow } from 'enzyme';
import 'jest-styled-components';

import {
  StyledArticlesSection,
  StyledCounterNews,
  StyledJumbotron,
  StyledHeader
} from '../styles';

describe('StyledArticlesSection', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<StyledArticlesSection />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe('StyledCounterNews', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<StyledCounterNews />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe('StyledJumbotron', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<StyledJumbotron />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe('StyledHeader', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<StyledHeader />);
    expect(wrapper).toMatchSnapshot();
  });
});
