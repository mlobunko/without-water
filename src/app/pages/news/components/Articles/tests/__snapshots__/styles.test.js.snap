// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledArticlesSection renders correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
}

<div
  className="c0"
/>
`;

exports[`StyledCounterNews renders correctly 1`] = `
.c0 {
  text-align: center;
}

<div
  className="c0"
/>
`;

exports[`StyledHeader renders correctly 1`] = `
.c0 {
  margin-bottom: 2rem;
}

<h3
  className="c0"
/>
`;

exports[`StyledJumbotron renders correctly 1`] = `
.c0 {
  max-width: 700px;
}

.c0.c0 {
  background-color: rgb(250,250,250);
}

<Jumbotron
  className="c0"
  tag="div"
/>
`;
